app/questionnaire/questionnaire_store_updater.py

```diff
@@ -15,21 +15,18 @@
 from app.utilities.types import DependentSection, LocationType
 
 
-class QuestionnaireStoreUpdater:
-    """Component responsible for any actions that need to happen as a result of updating the questionnaire_store"""
+class BaseQuestionnaireStoreUpdater:
```

Comment:
Minor, but would be clearer as `QuestionnaireStoreUpdateBase` to match our other base classes.